utils/find-wasm.js

```diff
@@ -0,0 +1,35 @@
+const util = require('util');
+const glob = util.promisify(require('glob'));
+const fs = require('fs');
+
+
+
+async function findWasm(target) {
+    let res = await glob(`**/${target}/*.wasm`, { 
+        ignore: '**/node_modules/**'
+    });
+    return res;
+}
+
+async function findWasmFile(wasmFile, target) {
+    if (!fs.existsSync(wasmFile)) {
+        if (wasmFile === './out/main.wasm') {
+            let res = await findWasm(target);
+            if (res.length == 0) {
+                console.error(`Wasm file ${wasmFile} does not exist and none could be found for target ${target}.`);
+                process.exit(1);
+            }
+            if (res.length > 1) {
+                console.error(`Multiple Wasm files found ${res.join(', ')} please pass one with --wasmFile=file`);
```

Comment:
This output seems less than ideal. What do each of the entries in `res` look like? Let's make sure they are relative paths, not absolute. And then, let's list each on a separate line, like this:

```js
                console.error(`Multiple Wasm files found: ${res.map(file => 
                  `\n  • ${file}`
                )}\n\nPlease pass one with --wasmFile=file`);
```